health/micro-ui/web/packages/modules/campaign-manager/src/pages/employee/NewAppConfiguration/NewDrawerFieldComposer.js

```
@@ -0,0 +1,508 @@
+  if (!isFieldVisible()) {
+    return null;
+  }
+
+  const getFieldValue = () => {
+    const bindTo = panelItem.bindTo;
+    return getFieldValueByPath(selectedField, bindTo, panelItem.defaultValue || "");
+  };
+
+  // Get localized field value for text fields
+  const fieldValue = getFieldValue();
+  const localizedFieldValue = useCustomT(fieldValue);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)